server/the-usher-openapi-spec.yaml

```diff
@@ -548,6 +548,32 @@ paths:
           $ref: '#/components/responses/InternalError'
 
   /personas/{persona_key}:
+    get:
+      'x-swagger-router-controller': 'personas/persona'
+      operationId: getPersona
+      summary: Get a Persona by persona_key
+      parameters:
+      - $ref: '#/components/parameters/personaKeyPathParam'
+      tags:
+        - Admin APIs
+      security:
+        - bearerAdminAuth: []
+      responses:
+        200:
+          description: Returns a persona
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Persona"
+                properties:
+                  tenantname:
```

Comment:
@k1ch  are we adding the tenant name to just the GET? Should we also add it to the list personas response to be consistent?

Reply:
@mikelax both `GET:/personas` and `GET:/personas/{persona_key}` include tenant name in the response for the convenience.